docs/architecture/how/resharding.md

```diff
@@ -1,37 +1,56 @@
-# Resharding - current state.
+# Resharding
+
+## Resharding
+
+Resharding is the process in which the shard layout changes. The primary purpose
+of resharding is to keep the shards small so that a node meeting minimum hardware
+requirements can safely keep up with the network while tracking some set minimum 
+number of shards. 
+
+## Specification
+
+The resharding is described in more detail in the following NEPs:
+* [NEP-0040](https://github.com/near/NEPs/blob/master/specs/Proposals/0040-split-states.md)
+* [NEP-0508](https://github.com/near/NEPs/pull/508) - TODO - once merged use the master link
```

Comment:
Leaving comment as TODO

Reply:
Sorry, what do you mean?

Reply:
Actually, please ignore this comment. I thought we would like to merge in the NEP before this PR but that doesn't make sense